### ISSUE TITLE
Normalization Bug in both training and testing

### ISSUE THREAD

**Issue by Chuyun-Shen** [GewelsJI/VPS#50]
### Bug Description
There is a tensor indexing bug in the normalization implementation that affects both training and testing. The Normalize class (in test script) and Normalize_video class (in training dataloader) use incorrect indexing that normalizes image columns instead of RGB channels.
### Location
Training code:
File: lib/dataloader/preprocess.py
Class: Normalize_video (lines 84-96)
Test code:
File: scripts/my_test.py (or test script)
Class: Normalize
### Root Cause
The pipeline order is:
toTensor_video() / ToTensor() → converts PIL Image to tensor with shape [C, H, W]
Normalize_video() / Normalize() → applies normalization
Current implementation:
class Normalize_video(object):
    def __call__(self, imgs, labels):
        res_img = []
        for img in imgs:  # img shape: [C, H, W]
            for i in range(3):
                img[:, :, i] -= float(self.mean[i])  # ❌ This indexes columns, not channels!
            for i in range(3):
                img[:, :, i] /= float(self.std[i])
            res_img.append(img)
        return res_img, labels
class Normalize_video(object):    def __call__(self, imgs, labels):        res_img = []        for img in imgs:  # img shape: [C, H, W]            for i in range(3):                img[:, :, i] -= float(self.mean[i])  # ❌ This indexes columns, not channels!            for i in range(3):                img[:, :, i] /= float(self.std[i])            res_img.append(img)        return res_img, labels
Problem:
img[:, :, i] on a [C, H, W] tensor selects the i-th column (width dimension), not the i-th channel
This normalizes only the first 3 columns of the image instead of the RGB channels

**Comment by GewelsJI** [comment]
Thanks for pointing out such bug, but this only affects the value overflow issue when W<3. So this might not largely hurt the finally training results, according to my experience, and even get better performance. 

Because the normalization statistics `Normalize([0.4732661, 0.44874457, 0.3948762], [0.22674961, 0.22012031, 0.2238305])` are calculated from ImageNet, this bug just equivalent with a small controlled and fixed perturbation on image data. 

Really suggest you directly use the raw implementation from PyTorch, like this

```
from torchvision.transforms import Normalize

transforms.Compose([
    Resize((config.size[0], config.size[1])),
    ToTensor(),
    transforms.Normalize(
        mean=[0.4732661, 0.44874457, 0.3948762],
        std=[0.22674961, 0.22012031, 0.2238305]
    )
])
```

Anyway, this is based on my previous experience, I have not verify it further. 

Let me know if you get updated results, we can talk further to make this codebase better.

Cheers.

**Comment by GewelsJI** [comment]
We plan to fix this bug and reverify the performance before March, stay tuned. Thanks, please reopen this issue anytime if you have any suggestions.
